src/python/pants/util/ordered_set.py

```diff
@@ -204,11 +204,11 @@ class FrozenOrderedSet(_AbstractOrderedSet[T_co], Hashable):
 
     def __init__(self, iterable: Iterable[T_co] | None = None) -> None:
         super().__init__(iterable)
-        self._hash: int | None = None
+        self.__hash: int | None = None
```

Comment:
Why?

Reply:
The built-in type has a _hash member apparently 🤷‍♂️